main.js

```diff
@@ -1190,6 +1191,7 @@ class Parcel extends utils.Adapter {
                 const sendungsObject = { id: sendung.id, name: sendung.sendungsinfo.sendungsname, status: status, source: "DHL" };
 
                 sendungsObject.inDelivery = this.inDeliveryCheck(sendungsObject);
+                sendungsObject.delivered = sendung.sendungsdetails.sendungsverlauf.fortschritt == sendung.sendungsdetails.sendungsverlauf.maximalFortschritt? true : false;
```

Comment:
You should check for the state otherwise the adapter will crash with property of null